frontend/src/Commons/Button/Button.style.ts

```diff
@@ -4,28 +4,27 @@ import screenSizes from '@Constants/ScreenSize';
 
 type StyledButtonProps = {
   size: string;
-  type: string;
   disabled: boolean;
+  radius: string;
+  actions: string;
 };
 
 const StyledButton = styled.button<StyledButtonProps>`
   display: flex;
   align-items: center;
   justify-content: center;
-  max-height: ${(props) => (props.size === 'sm' ? '27px' : props.size === 'md' ? '25px' : '40px')};
+  max-height: 42px;
   height: 100%;
   width: 100%;
-  max-width: ${(props) => (props.size === 'sm' ? '52px' : props.size === 'md' ? '200px' : '300px')};
-  background-color: ${(props) => (props.type === 'button' || props.type === 'submit'
+  max-width: ${(props) => (props.size === 'sm' ? '62px' : props.size === 'md' ? '159px' : '327px')};
```

Comment:
This is really just an anonymous function using a nested ternary as a return, this could simply be expanded into three `if` statements that return the value, or a switch case. It would be a more scalable solution going that path (What if I want an XXL Button?)
Really I saw @fergmac suggest it in a separate PR related to my team and immediately thought to myself "Oh how naive I have been". Combined we can all make our future better 😅

<sub>*Though I absolutely know where you got this nested ternary from, and I'm sure they've grown as a person*</sub>

Reply:
Yeah thay makes more sense. I could implement that.